packages/opds-web-client/package.json

```diff
@@ -12,10 +12,11 @@
   "license": "Apache-2.0",
   "scripts": {
     "lint": "tslint -c tslint.json src/*.ts src/*.tsx src/**/*.ts src/**/*.tsx src/**/**/*.ts src/**/**/*.tsx && sass-lint -c .sass-lint.yml -v -q",
-    "test": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
+    "test": "rm -rf lib npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js lib/__tests__/*.js lib/**/__tests__/*.js lib/**/**/__tests__/*.js",
+    "test-file": "npm run lint && tsc && cp -r src/stylesheets lib && mocha --require lib/testHelper.js",
     "dev": "webpack-dev-server --progress --history-api-fallback --hot --content-base build --port 8090 --config webpack.dev.config",
-    "prod": "NODE_ENV=production webpack -p --progress --display-modules --config webpack.prod.config && rm -rf dist/lib",
```

Comment:
Can you confirm that dist/lib is no longer being generated? I don't remember why this was here but don't want to be publishing it if it is still created.

Reply:
Maybe the `rm -rf lib` in prepublish handles it.

Reply:
The command in prepublish was added because I encountered overriding issues (maybe just my laptop?) but I added back the command to delete `dist/lib`. That folder will be gone now. Thanks for that catch.